src/m365/commands/login.ts

```diff
@@ -151,6 +197,88 @@ class LoginCommand extends Command {
     await this.initAction(args, logger);
     await this.commandAction(logger, args);
   }
+
+  private tokensForMultipleTenants(accessTokenValue: string | string[] | undefined, tenantValue: string | undefined): boolean {
```

Comment:
tenantValue sounds odd. Let's rename it to make it clearer what is it that we're expecting